DemoApp/app/screens/DataScreen.js

```diff
@@ -16,7 +16,7 @@ export default class DataScreen extends Component {
     tabBarIcon: () => <Image style={{ width: 24, height: 24 }} source={DataTabBarIcon} />,
     tabBarOnPress: ({ defaultHandler, navigation }) => {
       // Allow consequent presses to refresh the screen.
-      const refreshScreen = navigation.getParam('refreshData');
+      const refreshScreen = navigation.getParam('refreshScreen');
```

Comment:
For consistency with other screens, let's keep the param `refreshScreen` and rename https://github.com/microsoft/appcenter-sdk-react-native/pull/734/files#diff-5c743115ed0a0fcdd754f9eac490cb37R47.

Reply:
Do the same for DemoApp.

Reply:
The param was refreshData before and I changed it to refreshScreen since it was incorrect and getting no params.  Should I change the logic to put the params into refreshData?

Reply:
Yes, I meant keep `const refreshData = navigation.getParam('refreshData')` and change the logic to set param https://github.com/microsoft/appcenter-sdk-react-native/pull/734/files#diff-5c743115ed0a0fcdd754f9eac490cb37R47.